doc/source/atomgroup.rst

```diff
@@ -231,42 +222,45 @@ These methods return a user-ordered :class:`~MDAnalysis.core.groups.AtomGroup` t
 Empty AtomGroups
 ----------------
 
-Empty AtomGroups can be constructed from each method. For example, using selection language::
+Empty AtomGroups can be constructed from each method. For example, using selection language:
+
+.. ipython:: python
+
+    u.select_atoms("resname DOES_NOT_EXIST")
 
-    >>> u.select_atoms("resname DOES_NOT_EXIST")
-    <AtomGroup with 0 atoms>
+and indexing:
 
-and indexing::
+.. ipython:: python
+    
+    u.atoms[6:6]
+
+or set methods:
+
+.. ipython:: python
+    
+    u.atoms - u.atoms
 
-    >>> u.atoms[6:6]
-    <AtomGroup with 0 atoms>
+Creating an :class:`~MDAnalysis.core.groups.AtomGroup` from an empty list requires passing in a :class:`~MDAnalysis.core.universe.Universe`.
 
-or set methods::
+.. ipython:: python
 
-    >>> u.atoms - u.atoms
-    <AtomGroup with 0 atoms>
+    ag = mda.AtomGroup([], u)
+    ag
 
-Creating an :class:`~MDAnalysis.core.groups.AtomGroup` from an empty list requires passing in a :class:`~MDAnalysis.core.universe.Universe`. ::
+Empty AtomGroups have a length of 0 and evaluate to :code:`False` in a boolean context.
```

Comment:
I’d include how this can be used in analysis to skip empty AtomGroups (maybe resulting from distance selections that were too picky)